s_maxSize should be 'unsigned' rather than 'int' in WidthCache.h

Ahmad-S792 opened the pull request:
#### 315cbcb6456867e1bbad1a1031124d931b8b2914
<pre>
s_maxSize should be &apos;unsigned&apos; rather than &apos;int&apos; in WidthCache.h

<a href="https://bugs.webkit.org/show_bug.cgi?id=253272">https://bugs.webkit.org/show_bug.cgi?id=253272</a>

Reviewed by Myles C. Maxfield.

This patch is modifying &apos;s_maxSize&apos; to &apos;unsigned&apos; from &apos;int&apos; since other uses of
s_maxSize are comparison with other unsigned types.
Also update it to &apos;constexpr&apos; from &apos;const&apos;.

* Source/WebCore/platform/graphics/WidthCache.h:

Canonical link: <a href="https://commits.webkit.org/261252@main">https://commits.webkit.org/261252@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e4fa815a237a29c6fcc35bd9439c6a619644881c

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/111075 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/20216 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/43640 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/2503 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/119919 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/115033 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/21590 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/11318 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/86/builds/2143 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/116830 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/16033 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/99217 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/103582 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/97990 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/30882 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/44495 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/12732 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/32216 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/86401 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/13275 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/84/builds/9200 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/18691 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/62/builds/51814 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7803 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/15225 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

litherum commented on Source/WebCore/platform/graphics/WidthCache.h:
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2012 Apple Inc. All rights reserved.
+ * Copyright (C) 2012-2023 Apple Inc. All rights reserved.
```
Are you an Apple employee? I don't think this is right.

Ahmad-S792 replied:
Will remove in next commit and add merge-queue. :-)

litherum reviewed (approved):
r=me modulo the copyright line

Ahmad-S792 committed:
s_maxSize should be 'unsigned' rather than 'int' in WidthCache.h

https://bugs.webkit.org/show_bug.cgi?id=253272

Reviewed by Myles C. Maxfield.

This patch is modifying 's_maxSize' to 'unsigned' from 'int' since other uses of
s_maxSize are comparison with other unsigned types.
Also update it to 'constexpr' from 'const'.

* Source/WebCore/platform/graphics/WidthCache.h:

Canonical link: https://commits.webkit.org/261252@main
